src/append/rolling_file/rolling.rs

```diff
@@ -282,7 +277,11 @@ impl State {
 
                 if self.log_filename_prefix.is_none()
                     && self.log_filename_suffix.is_none()
-                    && Date::parse(filename, &self.date_format).is_err()
+                    && Zoned::strptime(
+                        format!("{} %:z", self.date_format),
+                        format!("{filename} +00:00"),
+                    )
+                    .is_err()
```

Comment:
Quite a bit hacky ... But doesn't seem other better way.

Possible `date_format` are:

```
            Rotation::Minutely => "%F-%H-%M",
            Rotation::Hourly => "%F-%H",
            Rotation::Daily => "%F",
            Rotation::Never => "%F",
```

Reply:
I don't understand what you're trying to do here. Are you just trying to verify a date format? If so, can you use `jiff::civil::Date::strptime(&self.date_format, filename).is_err()`? Every datetime type in Jiff has `strptime` and `strftime` methods.

Reply:
@BurntSushi Your alternative is much better. Thank you!